### PR TITLE
Standardize Jenkinsfiles - Closes #1219

### LINKED ISSUE #1219: Standardize Jenkinsfiles
Child of #672 
E.g. using generic names for stages/steps. Should also cover setting up Snyk etc.

### PR DESCRIPTION
**Pull request by willclarktech**
### What was the problem?

There were some inconsistencies in the Jenkinsfiles. Also there was no automated check for vulnerabilities in dependencies.

### How did I fix it?

I tidied up the existing Jenkinsfiles and added new files for the automated check. Note the node used in the new files was a guess and will almost certainly have to be changed before merging.

### How to test it?

Check Jenkins.

### Review checklist

* The PR solves #1219 
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)

### CODE REVIEW

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -0,0 +1,23 @@
+pipeline {
+	agent { node { label 'node-08' } }
```
Comment (fchavant) [inline Jenkinsfile.nightly]: We do not have any node matching `node-08` at the moment.
If we add more nodes/labels can we maybe chose more explicit names?

File: Jenkinsfile.private [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		stage('Install dependencies') {
+			steps {
+				sh 'npm install --verbose'
+			}
+		}
+		stage('Run vulnerabilities check') {
+			steps {
+				withCredentials([string(credentialsId: 'liskhq-snyk-token', variable: 'SNYK_TOKEN')]) {
+					sh '''
+					if [ -r ~/.lisk-template.package.json.sha1sum ]; then
```
Comment (fchavant) [inline Jenkinsfile.private]: Should be `~/.lisk.package.json.sha1sum`, not `~/.lisk-template.package.json.sha1sum`

File: Jenkinsfile.private [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+						if sha1sum --status --check ~/.lisk-template.package.json.sha1sum; then
```
Comment (fchavant) [inline Jenkinsfile.private]: Same as above.

File: Jenkinsfile.nightly (hunk 1 above)
Comment (willclarktech) [reply Jenkinsfile.nightly]: Definitely in favour of that. `dependency-check`? Or should I reuse an existing node?

Comment (fchavant) [reply Jenkinsfile.nightly]: `dependency-check` sounds good

**Pull request merged**